modules/prompt/functions/async

```diff
@@ -1 +1 @@
-../external/pure/async.zsh
\ No newline at end of file
+../external/async/async.zsh
```

Comment:
@indrajitr yeap, I had to manually fix it:
```
~/.zgen/sorin-ionescu/prezto-master/modules/prompt/functions ❯ mv async async.bak && ln -s ../external/pure/async.zsh async
```
My case could be an exception, because there are lot of moving parts involved here - `zgen`, `prezto`, `async`, and finally `prompt pure`